desktop_version/src/Font.h

```diff
@@ -56,6 +56,7 @@
 #define PR_CJK_HIGH (2 << 20) /* larger fonts should stick out fully on the top */
 #define PR_RTL_FORCE (1 << 22) /* force the RTL flag, not needed if the font is set to INTERFACE or LEVEL */
 #define PR_RTL_XFLIP (1 << 23) /* in RTL languages, mirror the X axis, so left is 320 and right is 0, and invert the meaning of PR_LEFT and PR_RIGHT */
+#define PR_FONT_CUSTOM (1 << 24) /* with PR_FONT_IDX, mark that the font index is of a custom font */
```

Comment:
This name is a little confusing, since we now have the following under `PR_FONT_*`:
- `PR_FONT_INTERFACE`
- `PR_FONT_LEVEL`
- `PR_FONT_8X8`
- `PR_FONT_IDX(idx, rtl)`
- `PR_FONT_CUSTOM`

It reads like it's just another possible font option. Maybe `PR_FONT_IDX_IS_CUSTOM`?